cirq-core/cirq/sim/clifford/act_on_stabilizer_ch_form_args.py

```diff
@@ -12,34 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Any, Dict, TYPE_CHECKING, List, Sequence
+from typing import Any, Dict, List, Optional, Sequence
+from cirq.sim.clifford.clifford_simulator import CliffordState
```

Comment:
The standard format in cirq is to import modules rather than classes. So `from cirq.sim.clifford import clifford_simulator`, and use `clifford_simulator.CliffordState` in the code

Reply:
Thanks for the detailed instruction.

Reply:
Thanks, you can resolve this one.